admin-tools/data_mover.sh

```diff
@@ -0,0 +1,116 @@
+#! /bin/bash
```

Comment:
This documentation should describe the "failover" procedure in the case that the script fails due to an unexpected interruption while it is working (if the case of migrating large data sets, this may happen, e.d. due to a connectivity lost), e.g. clean all the stuff in the dst cluster and run the script again.